packages/fast-components-styles-msft/src/text-field/index.ts

```diff
@@ -0,0 +1,45 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { ITextFieldClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import { applyType } from "../utilities/typography";
+import * as Chroma from "chroma-js";
+
+function applyMixedColor(mixValue?: number, alpha?: number): (config: IDesignSystem) => string {
+    return function applyColors(config: IDesignSystem): string {
+        if (alpha) {
+            return Chroma.mix(config.foregroundColor, config.backgroundColor, mixValue).alpha(alpha).css();
+        } else {
+            return Chroma.mix(config.foregroundColor, config.backgroundColor, mixValue).css();
+        }
+    };
+}
+
+const styles: ComponentStyles<ITextFieldClassNameContract, IDesignSystem> = {
+    textField: {
+        ...applyType("t7", "vp1"),
+        color: applyMixedColor(0.46),
+        fontWeight: "100",
```

Comment:
We have a utility for font weights that we should use

Reply:
JSS expecting fontWeight as a string. Should utility be updated to reflect this or should it be converted in the components handling of that property?

Reply:
Interesting. Not really in the scope of the PR but we could either simply `toString` the functions' results, or change it to return a string. I wonder if it is being used elsewhere and needs to be a number in that case.

Reply:
we may just need to update the typings for a stylesheet - I think right now it explicitly expects a string but sometimes numbers are valid.